src/main/java/impl/weather/impl/YearTemperatureStatsParserImpl.java

```diff
@@ -0,0 +1,24 @@
+package impl.weather.impl;
+
+import api.weather.YearTemperatureStats;
+import api.weather.YearTemperatureStatsParser;
+
+import java.time.Month;
+import java.util.Collection;
+
+public class YearTemperatureStatsParserImpl implements YearTemperatureStatsParser {
+    @Override
+    public YearTemperatureStats parse(Collection<String> rawData) {
```

Comment:
On a second though maybe we'd want to validate the argument...

Reply:
We shouldn't. Correct input is guaranteed